recipes/pciutils/all/conanfile.py

```diff
@@ -0,0 +1,76 @@
+import os
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+
+
+class PciUtilsConan(ConanFile):
+    name = "pciutils"
+    license = "BSD-3-Clause"
+    description = "The PCI Utilities package contains a library for portable access to PCI bus"
+    topics = ("pci", "pci-bus", "hardware", "local-bus")
+    homepage = "https://github.com/pciutils/pciutils"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False], "with_zlib": [True, False], "with_udev": [True, False]}
+    default_options = {"shared": False, "fPIC": True, "with_zlib": True, "with_udev": False}
+
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def configure(self):
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration("Platform {} is currently not supported by this recipe".format(self.settings.os))
+
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def requirements(self):
+        if self.options.with_zlib:
+            self.requires("zlib/1.2.11")
+        if self.options.with_udev:
+            # TODO: Enable libudev option when available
+            raise ConanInvalidConfiguration("libudev requires conan-io/conan-center-index#2468")
+            self.requires("systemd/system")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        tools.rename(extracted_dir, self._source_subfolder)
+
+    def _make(self, targets):
+        yes_no = lambda v: "yes" if v else "no"
+        fpic = "-fPIC" if self.options.get_safe("fPIC") else ""
+        autotools = AutoToolsBuildEnvironment(self)
+        autotools.make(args=["SHARED={}".format(yes_no(self.options.shared)),
+                             "ZLIB={}".format(yes_no(self.options.with_zlib)),
+                             "HWDB={}".format(yes_no(self.options.with_udev)),
+                             "PREFIX={}".format(self.package_folder),
+                             "OPT={}".format(fpic),
```

Comment:
Looking at some logs (e.g. [Release,gcc7,linux](https://c3i.jfrog.io/c3i/misc/logs/pr/3758/5/pciutils/3.7.0/e7bf0619084c8cebdf6c8601a09317fb3ad094b2/create_stdout.txt)), the compiler flags do not contain `-m64 -O2` like it should according to the default `AutotoolsBuildEnvironment` object.

From https://github.com/pciutils/pciutils/blob/e12bd01eea67ca8cf539263124843ba281eb6ecc/Makefile#L5
I think CFLAGS and CPPFLAGS of the AutotoolsBuildEnvironment helper object has the fPIC option set + other flags (possibily overriden).

This is an **untested** suggestion.

```suggestion
        autotools = AutoToolsBuildEnvironment(self)
        autotools.make(args=["SHARED={}".format(yes_no(self.options.shared)),
                             "ZLIB={}".format(yes_no(self.options.with_zlib)),
                             "HWDB={}".format(yes_no(self.options.with_udev)),
                             "PREFIX={}".format(self.package_folder),
                             "OPT={}".format("{} {}".format(autotools.vars["CPPFLAGS"], autotools.vars["CFLAGS"])),
```

Reply:
Great catch! Too a small tweak but it makes a difference!

Reply:
By accident :smile: . But this change makes the difference between building with or without debug symbols :smile: 
That would be a nice test by the cci hooks btw.
Test, when `build_type` is in `("Debug", "RelWithDebInfo")`, whether debug information is actually available in the libraries/executables..